src/crimson/os/seastore/cache.h

```diff
@@ -336,6 +336,7 @@ class Cache {
       SUBTRACE(seastore_cache,
           "{} {}~{} is present in cache -- {}",
           T::TYPE, offset, length, *cached);
+      ceph_assert(cached->is_fully_loaded());
```

Comment:
Hmm, why?

IIUC the not-fully-loaded extents can be in both Cache (as CLEAN) and Transaction (as CLEAN_EXISTING), right?

Reply:
All not-fully-loaded extents no matter in transaction or cache will be loaded when `get_extent_viewable_by_trans` before `pin_to_extent` at TM method `read_pin()`, so I think no not-fully-loaded extents will run into here

Reply:
Added not-fully-loaded case in get_extent in latest push to ensure get_extent can handle not-fully-loaded extents.